eve/commands/cmd_crypto.py

```diff
@@ -0,0 +1,35 @@
+import click
+import locale
+from eve.service import svc_crypto
+from eve.config import DEFAULT_COIN, CONVERT_TO_CURRENCY
+from eve.utilities import formatting as f
+
+
+class Context:
+    def __init__(self, coin, destination_currency):
+        self.coin = coin.upper()
+        self.destination_currency = destination_currency.upper()
+        self.crypto = svc_crypto.Crypto()
+
+
+@click.group()
+@click.option("-c", "--coin", type=str, help="Coin", default=DEFAULT_COIN, show_default=True)
+@click.option("-d", "--destination-currency", type=str, help="destination", default=CONVERT_TO_CURRENCY, show_default=True)
```

Comment:
What are your thoughts about changing this variable to `@click.option("-ccy", "--currency", type=str, help="Currency", ...)` - I feel like it is slightly more intuitive?

Reply:
Totally, `--destination-currency` is way too long